projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -290,10 +261,9 @@ export function NewsletterAccessDocumentSettings( {
 	const accessLabel = accessOptions[ _accessLevel ]?.label;
 
 	// Immediately close the dropdown dialog after setting the post meta.
-	const setPostMetaAndClose = onClose => {
+	const setPostMetaAndClose = () => {
```

Comment:
Maybe we could rename this as it does not close anymore?

Reply:
We could also directly use setPostMeta in the onChange?

Reply:
Fixed in [48777e7](https://github.com/Automattic/jetpack/pull/31224/commits/48777e70de1659ef0fa8ff3a6c274f166fe90258)